database/factories/UserFactory.php

```diff
@@ -16,12 +17,20 @@
 |
 */
 
-$factory->define(User::class, function (Faker $faker) {
-    return [
-        'name' => $faker->name,
-        'email' => $faker->unique()->safeEmail,
-        'email_verified_at' => now(),
-        'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
-        'remember_token' => Str::random(10),
-    ];
-});
+$factory->define( \App\User::class, function ( Faker $faker ) {
```

Comment:
We do not need this factory yet. Please remove this file.

Reply:
Fixed.